test/mocha/release.js

```diff
@@ -8,7 +8,7 @@ function run(command, args, done) {
         process.stdout.write("\0");
     }, 5 * 60 * 1000);
     spawn(command, args, {
-        stdio: "ignore"
+        stdio: [ "ignore", 1, 2 ]
```

Comment:
Am I correct in assuming you want to ignore both `stdout` and `stderr` instead of just `stdout`?